osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -58,9 +57,11 @@ public ScoreTable()
         }
 
         /// <summary>
-        /// The statistics that appear in the table, in order of appearance.
+        /// The statistics that appear in the table, grouped by their display name. If multiple HitResults have the same
+        /// DisplayName (for example, "slider end" is the name for both HitResult.SliderTailTick and HitResult.SmallTickHit
+        /// in osu!std) the name will only be listed once.
```

Comment:
`osu!std` is not acknowledged as a term of reference for osu! ruleset. And one of the hit result references is wrong.

```diff
diff --git a/osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs b/osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs
index 671b5df33b..e6052b0e3b 100644
--- a/osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs
+++ b/osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs
@@ -58,8 +58,8 @@ public ScoreTable()
 
         /// <summary>
         /// The statistics that appear in the table, grouped by their display name. If multiple HitResults have the same
-        /// DisplayName (for example, "slider end" is the name for both HitResult.SliderTailTick and HitResult.SmallTickHit
-        /// in osu!std) the name will only be listed once.
+        /// DisplayName (for example, "slider end" is the name for both <see cref="HitResult.SliderTailHit"/> and <see cref="HitResult.SmallTickHit"/>
+        /// in osu!) the name will only be listed once.
         /// </summary>
         private readonly List<(LocalisableString displayName, IEnumerable<HitResult> results)> statisticResults = new List<(LocalisableString displayName, IEnumerable<HitResult> results)>();
 
```